docs/tutorials/google/start.ipynb

```diff
@@ -0,0 +1,559 @@
+        "  auth.authenticate_user(clear_output=False)\n",
+        "\n",
+        "print(\"Getting OAuth2 credentials.\")\n",
+        "print(\"Press enter after entering the verification code.\")\n",
+        "setup_auth()\n",
+        "print(\"Authentication complete.\")\n",
+        "print(\"installing cirq\")\n",
+        "!pip install cirq --quiet"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Getting OAuth2 credentials.\n",
+            "Press enter after entering the verification code.\n",
+            "Go to the following link in your browser:\n",
+            "\n",
+            "    https://accounts.google.com/o/oauth2/auth?client_id=32555940559.apps.googleusercontent.com&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&scope=openid+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fappengine.admin+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcompute+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Faccounts.reauth+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fdrive&code_challenge=7ckPGGeMO4VR06QLBmW1gbAGBU17P8D6F3AqB0ZRGc4&code_challenge_method=S256&access_type=offline&response_type=code&prompt=select_account\n",
+            "\n",
+            "\n",
+            "Enter verification code: ··········\n",
+            "Authentication complete.\n",
+            "installing cirq\n",
+            "\u001b[K     |████████████████████████████████| 1.4MB 4.5MB/s \n",
+            "\u001b[?25h"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ocy3JossWU57",
+        "colab_type": "text"
+      },
+      "source": [
+        "**Authentication details** Double clicking on the project_id block above should expose the code that is run when you run this code block.  This code uses [colabtools](https://github.com/googlecolab/colabtools/blob/master/google/colab/auth.py#L123) auth module to ensure that \"Application Default Credentials\" are set and then creates a variable `colab_auth` which can be used in Cirq to authenticate your calls to Quantum Engine.\n",
+        "\n",
+        "If you are going to run code outside of colab and want to authenticate, see the below section on running from the command-line."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "47n3bsx3Nl9g",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Create a circuit\n",
+        "Now that you've enabled Quantum Engine and configured the notebook, let's create a basic program with Cirq. After reviewing the code, **run this block** to run a circuit, and print a circuit diagram and results. To learn more, refer to the [Intro to Cirq](http://go/intro-to-cirq) and [Cirq documentation](https://cirq.readthedocs.io)."
```

Comment:
Replace `go/intro-to-cirq` with external link.

Reply:
Done.